Integrations/integration-Rapid7_Nexpose.yml

```diff
@@ -1445,6 +1499,31 @@ script:
       description: Common Vulnerabilities and Exposures ids
     description: Returns the details and possible remediations for an asset's given
       vulnerability.
+  - name: nexpose-create-site
+    arguments:
+    - name: name
+      required: true
+      description: <string> The site name. Name must be unique.
+    - name: description
+      description: <string> The site's description.
+    - name: assets
+      required: true
+      description: Multiple criteria of <string> Specify asset addresses to be included
+        in site scans
+      isArray: true
+    - name: scanTemplateId
+      description: <string> The identifier of a scan template. Default scan template
```

Comment:
where I can get this scanTemplateId, any specific command should I run?

Reply:
yes, `nexpose-get-report-templates`, I will add to the description